operations/deployment/terraform/security-group.tf

```diff
@@ -8,7 +8,10 @@ resource "aws_security_group" "ec2_security_group" {
     cidr_blocks = ["0.0.0.0/0"]
   }
   tags = {
-    Name = "${var.aws_resource_identifier}"
+    merge(
+      Name = "${var.aws_resource_identifier}",
+      var.additional_tags
+  )
```

Comment:
I think this merge can be removed because the additional tags are set in the `default_tags` here:
https://github.com/bitovi/github-actions-deploy-docker-to-ec2/pull/17/files#diff-088d9a58b5e5c54a030a57c8526a5a9659ede6fb83fb4e6a2eb7ade6f38600b0R39